scss/product/components/_badge.scss

```diff
@@ -90,4 +94,15 @@
   display: inline-flex;
   align-items: center;
   justify-content: center;
-}
\ No newline at end of file
+}
+
+// notification with a dot
+ .badge-notification-dot {
```

Comment:
Can you share the screenshot of badge-notification-dot?

Reply:
i think we can move this style to sidebar.scss right? it wont make any sense adding it in badge.scss @Pallavi-PH